e2e/snapshots.js

```diff
@@ -1,16 +1,16 @@
 require('dotenv').config({ override: true })
 const process = require('node:process')
 
-const snapShotFactory = (name, path, overrides) => {
-    const baseUrl = process.env.PERCY_BRANCH === 'production' ? 'https://design.energysage.dev' : 'http://localhost:8500';
-    const url = `${baseUrl}${path}`;
+function snapShotFactory(name, path, overrides) {
```

Comment:
It's interesting the favoring of `function` over unnamed functions assigned to a variable